webviews/tsconfig.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
Can we name this file to something else? Like I said, the original code has these things messed up badly, we could do better.
`webview_config.json`?

Reply:
I am not sure about it. Maybe the system(node) uses the name internally to recognize the configuration file because I haven't used the file name explicitly anywhere while developing files. I need to do more research on it to answer this.